package.json

```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "husky": "^4.3.6",
+    "node-sass": "^5.0.0",
```

Comment:
could you go with the dart-sass that we have in flibs?

Reply:
Build is failing with error, trying to fix it, it doesn't happen for me locally: https://github.com/infinum/eightshift-forms/runs/2663211028?check_suite_focus=true#step:9:8. Tho now that I think about it, there might be another way to fix it.

Reply:
Just updated all packages + nuked package.lock, should be ok now (since `sass` is a dependency of `frontend-libs`).